src/components/footer/footer.yaml

```diff
@@ -4,6 +4,9 @@ params:
   required: false
   description: Object containing options for the meta navigation.
   params:
+  - name: title
+    type: string
+    description: Title for a meta item section
```

Comment:
Should we add something in the description about this title being hidden?

If we choose to keep a default around, we'd also want to say that this overrides the default title for the meta item section...

Reply:
`params.meta.visuallyHiddenText` or `params.meta.visuallyHiddenTitle` would be consistent with the approach we use elsewhere.

Reply:
I went with visuallyHiddenTitle, I've not updated the description, as I think the parameter name does that, but have mentioned the default.